setup/scripts/toRDF.js

```diff
@@ -47,20 +47,39 @@ const toRDF = async (jsonURL, xmlURL) => {
     const decodeHtmlEntities = (text) => text.replace(/&#(\d+);/g, (match, dec) => String.fromCharCode(dec))
 
     // Helper function to provide information for <skos:changeNote> based on what the xml data looks like
+    // It also adds attributes for easier parsing
     const createChangeNote = (date, userId, userNote, changeNoteItems) => {
-      let changeNoteText = `${date} [${userId}] ${userNote}`
-      if (changeNoteItems) {
-        const changeNoteItem = Array.isArray(changeNoteItems.changeNoteItem)
-          ? changeNoteItems.changeNoteItem
-          : [changeNoteItems.changeNoteItem]
-        changeNoteItem.forEach((item) => {
-          const { '@_systemNote': systemNote, '@_newValue': newValue } = item
-          const decodedNewValue = decodeHtmlEntities(newValue || '').trim()
-          changeNoteText += `\n${systemNote} (${decodedNewValue})`
-        })
-      }
-
-      return changeNoteText
+      let changeNoteText = 'ChangeNote Information\n \n'
+      changeNoteText += `date: ${date}\n`
```

Comment:
I'm wondering if we should change the field names to be more human readable, i.e., User Id:, User Note:, New Value:, Old Value:, etc. Other things like Change Note Information, Change Note Item #1, etc.